### ISSUE TITLE
Black screen

### ISSUE THREAD

**Issue by scalp42** [ShadowOne333/A-Link-to-the-Past-Redux#8]
Hi @ShadowOne333,

Thanks a lot for putting all this together. Unfortunately, I keep running into a black screen, using RetroArch with Snes9x 1.61 (tried on a Miyoo Mini as well).

I'm using https://www.romhacking.net/patch/ to patch (but I've tried other tools as well) and no dice.

I can confirm the MD5 is correct: `608c22b8ff930c62dc2de54bcd6eba72` and I've only applied `Link to the Past Redux.ips` per instructions (no optional patches added yet).

Final MD5 when ROM is patched with just `Link to the Past Redux.ips` is `69ac9b4587fba461c3f776d29b4a46c3` if by any chance you could confirm.

MD5 for `Link to the Past Redux.ips` is `8912ce507e71b96654cb73a517907fc1` (using the latest release from GitHub, [v10.1](https://github.com/ShadowOne333/A-Link-to-the-Past-Redux/releases/tag/v10)).

I'm out of ideas, any help would be greatly appreciated 🙏

![Screen Shot 2022-12-15 at 10 30 26](https://user-images.githubusercontent.com/1475276/207941392-2c72af88-433f-4964-8bc5-8c679ded92fe.png)

**Comment by scalp42** [comment]
﻿﻿Here's a screenshot for confirmation as well:

![Screen Shot 2022-12-15 at 10 30 26](https://user-images.githubusercontent.com/1475276/207940801-030d4968-bca2-4fd9-8af4-efeb0145e4cf.png)

**Comment by ShadowOne333** [comment]
Most likely you enabled the "Add temporary header" option on the online patcher.
I tried patching a clean ROM with the online patcher, and the patched ROM works fine on my end.

The final MD5 checksum for the patched ROM with only Redux patched is: `5e6da17ada2a9d85cef3263fcd2de658`

This is the data from the online patcher:
![image](https://user-images.githubusercontent.com/13888698/207943876-38a8a506-f846-436e-ac79-ea9af6f80bff.png)

Make sure you have the exact same data from the clean ROM and that you only use "Link to the Past Redux.ips".

**Comment by scalp42** [comment]
Thanks for the quick answer! I didn't enable the header but let me try again from scratch and I'll get back to you 🙌

**Comment by scalp42** [comment]
The plot thickens:

```
MD5 (Legend of Zelda, The - A Link to the Past (USA).sfc) = 608c22b8ff930c62dc2de54bcd6eba72
MD5 (github.sfc) = 69ac9b4587fba461c3f776d29b4a46c3
MD5 (romhacking.sfc) = 5e6da17ada2a9d85cef3263fcd2de658
```

When using package from https://www.romhacking.net/reviews/8127, it works and I get the same MD5 as you.

When using the latest GitHub release, I get a black screen and a different MD5 hash 😅

**Comment by ShadowOne333** [comment]
That's expected. The GitHub patch is made with the source code in this repository, while the Romhacking.net patch was made still using the old method/tools of binary hacking.

I patched a clean ROM with the GitHub patch just now, get the same MD5 sum you posted and it runs fine on Snes9x for me, so both the Romhacking and GitHub patches work on my end when patched in a clean ROM.

**Comment by scalp42** [comment]
Thanks for the quick feedback, appreciate the help.

I've tried many times to be sure with the latest GitHub release. I always end up with the MD5 for the patched ROM `69ac9b4587fba461c3f776d29b4a46c3` and I just can't get it to work across different combinations of platform/cores so I have no idea 😭

**Comment by ShadowOne333** [comment]
I recommend deleting the save file too, not sure if that causes issues. I'm running Snes9x 1.61.
But either way that's weird, you could attempt with the file I uploaded, maybe that helps.

[Zelda3-Redux.zip](https://github.com/ShadowOne333/A-Link-to-the-Past-Redux/files/10240064/Zelda3-Redux.zip)

**Comment by scalp42** [comment]
I didn't see your answer in time but you're right, I think it was trying to load a state not compatible. Nuked everything and I was greeted by the title sound!

Thank you again for the help @ShadowOne333, much appreciated ❤️

**Comment by ShadowOne333** [comment]
Awesome! Glad to know the issue got resolved. Enjoy the hack!
